### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Added `owners` table
- Inserted more data to be used for the performance audit
- Tested the estimation time of execution using `EXPLAIN ANALYZE`
- Used `INDEX` to reduce the execution time
- Took screenshots of results to show the reduction in execution time
- Added relevant commands to the respective files in the repo

#### - Before:
before animalid query
#### - After:
after animalid query
#### - Before:
before vetid query
#### - After:
after vetid query
#### - Before:
before email query
#### - After:
after email query

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +110,6 @@ I would like to thank my colleagues at Microverse for their support during the p
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT) licensed.
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly fix your MIT license link because it is taking me to a 404 page.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Uthmanbello) [reply README.md]: @fobadara,
Ok. Thanks

**Review by andersonlebon — APPROVED** [review]
## Hi @Uthmanbello 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Uthmanbello** [comment]
@andersonlebon,
Thanks 😃

**Pull request merged**